images/etcd/configs/latest.apko.yaml

```diff
@@ -0,0 +1,16 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  packages:
+    - etcd
+    - ca-certificates-bundle
+    - glibc
```

Comment:
We shouldn't need to specify glibc.

Reply:
Double checked and we do need to keep glibc here - the builds are dynamic on purpose.